### PR TITLE
Add emrcontainers service

### PR DESCRIPTION
**Pull request by shuheiktgw**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates https://github.com/hashicorp/terraform-provider-aws/issues/16717

Release note for [CHANGELOG](https://github.com/hashicorp/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
None
```

Output from acceptance testing:

```
None
```
For the first step to support emrcontainers, I added an emrcontainers service. Thank you for your review! 👍

### CODE REVIEW

File: aws/config.go [hunk 1]
```diff
@@ -267,6 +268,7 @@ type AWSClient struct {
 	elbconn                             *elb.ELB
 	elbv2conn                           *elbv2.ELBV2
 	emrconn                             *emr.EMR
+	emrcontainerscon                    *emrcontainers.EMRContainers
```
Comment (bflad) [inline aws/config.go:271]:
```suggestion
	emrcontainersconn                   *emrcontainers.EMRContainers
```

File: aws/config.go [hunk 2]
```diff
@@ -507,6 +509,7 @@ func (c *Config) Client() (interface{}, error) {
 		elbconn:                             elb.New(sess.Copy(&aws.Config{Endpoint: aws.String(c.Endpoints["elb"])})),
 		elbv2conn:                           elbv2.New(sess.Copy(&aws.Config{Endpoint: aws.String(c.Endpoints["elb"])})),
 		emrconn:                             emr.New(sess.Copy(&aws.Config{Endpoint: aws.String(c.Endpoints["emr"])})),
+		emrcontainerscon:                    emrcontainers.New(sess.Copy(&aws.Config{Endpoint: aws.String(c.Endpoints["emrcontainers"])})),
```
Comment (bflad) [inline aws/config.go:512]:
```suggestion
		emrcontainersconn:                   emrcontainers.New(sess.Copy(&aws.Config{Endpoint: aws.String(c.Endpoints["emrcontainers"])})),
```

**Review by bflad — APPROVED** [review]
Typo fixes, otherwise looks good to me, thank you @shuheiktgw 🚀

**Comment by shuheiktgw** [comment]
Sorry for the typos... Thank you for your review, @bflad!

**Pull request merged**